proto/bigtable.d.ts

```diff
@@ -1,4 +1,5 @@
 import * as $protobuf from "protobufjs";
+import * as long from "long";
```

Comment:
This doesn't work, we'll need to remove the `-i` parameter and unfortunately hardcode the correct import for now.

```ts
import * as Long from 'long';
```

Reply:
Yikes okay :(